__init__.py

```diff
@@ -35,12 +35,17 @@
     from bpy.props import *
     from bpy_extras.io_utils import (
             ExportHelper,
+            orientation_helper_factory,
+            axis_conversion
             )
 
     from . import blendergltf
 
+    GLTFOrientationHelper = orientation_helper_factory(
+        "GLTFOrientationHelper", axis_forward='-Z', axis_up='Y'
+    )
 
-    class ExportGLTF(bpy.types.Operator, ExportHelper):
+    class ExportGLTF(bpy.types.Operator, ExportHelper, GLTFOrientationHelper):
```

Comment:
It doesn't seem to work using composition / initializing it like the other properties. I remember I looked at the .obj exporter for reference and it also inherits.